samples/excel/30-events/events-comment-event-handler.yaml

```
@@ -0,0 +1,195 @@
+order: 13
+id: excel-events-comment-event-handler
```

Consider changing the ID to better match the other event sample IDs.
```suggestion
id: excel-events-comments
```
(Note that if you make this change, you'll need to edit the excel mapping file too.)